target/cheri-common/cheri_utils.h

```diff
@@ -155,14 +174,26 @@ static inline bool cap_is_sealed_with_type(const cap_register_t *c)
         cheri_debug_assert(cap_get_otype_unsigned(c) <=
                            CAP_MAX_REPRESENTABLE_OTYPE);
     }
+#ifdef TARGET_AARCH64
+    /*
+     * Morello has positive signed otypes.
+     */
+    return cap_get_otype_unsigned(c) > CAP_LAST_NONRESERVED_OTYPE;
```

Comment:
This doesn't look right, the last non-reserved otype is 2^N-1 on Morello, unless this is a misnomer that should be fixed?

Reply:
I think its a misnomer because of the reversal of which end of the type space has special types between morello and other platforms. The name of this field existed prior to morello.